src/kili/core/graphql/graphql_client.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         self._gql_transport = RequestsHTTPTransport(
             url=endpoint,
             headers=self._get_headers(),
-            timeout=30,
+            timeout=60,
```

Comment:
If we want to make this configurable, I would rather add an argument to the kili contructor instead of using an env var.
I don't think we should make this configurable, it really depends on our API response times and is not really related to something the user can do.